dev_environment/setup_dev_env.sh

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env bash
```

Comment:
I'd recommend renaming this script to setup_linux_dev_env.sh for clarity since in the future we may want a separate setup_windows_dev_env script

Reply:
Add the comment and change the script file accordingly